### PR TITLE
Update identity scaffolding dependency content

### PR DESCRIPTION
**Pull request by MackinnonBuck**
Updates dependencies of the "Bootstrap4" and "Bootstrap5" variants of Identity MVC scaffolding. This PR also adds new NPM scripts to make the dependency update process easier.

Validated the changes locally by installing and running the `aspnet-codegenerator` tool:
image

### CODE REVIEW

File: test/Scaffolding/E2E_Test/Compiler/Resources/SharedViews/_ValidationScriptsPartial.cshtml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ﻿<environment names="Development">
     <script src="~/lib/jquery-validation/dist/jquery.validate.js"></script>
-    <script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js"></script>
+    <script src="~/lib/jquery-validation-unobtrusive/dist/jquery.validate.unobtrusive.js"></script>
 </environment>
 <environment names="Staging,Production">
-    <script src="https://ajax.aspnetcdn.com/ajax/jquery.validate/1.17.0/jquery.validate.min.js"
+    <script src="https://ajax.aspnetcdn.com/ajax/jquery.validate/1.19.3/jquery.validate.min.js"
```
Comment (javiercn) [inline test/Scaffolding/E2E_Test/Compiler/Resources/SharedViews/_ValidationScriptsPartial.cshtml:6]: This is 1.21.0 https://www.npmjs.com/package/jquery-validation

Comment (javiercn) [reply test/Scaffolding/E2E_Test/Compiler/Resources/SharedViews/_ValidationScriptsPartial.cshtml:6]: It's what's on the other templates and I think the identity UI has either 1.21.0 or 1.20.0. We need to update the integrity shas

Comment (MackinnonBuck) [reply test/Scaffolding/E2E_Test/Compiler/Resources/SharedViews/_ValidationScriptsPartial.cshtml:6]: Version 1.19.3 is the latest version where that URL doesn't 404 (this file is used in E2E testing only).

@deepchoudhery, do you know what the process is like to get the latest version of `jquery-validation` on `https://ajax.aspnetcdn.com`?

Comment (deepchoudhery) [reply test/Scaffolding/E2E_Test/Compiler/Resources/SharedViews/_ValidationScriptsPartial.cshtml:6]: not quite no, these tests are quite out of date and most of them are skipped afaik. I would just match this file to the one shipping with Identity for now (bootstrap 5?). Will be writing a bunch of E2E for dotnet-scaffold soon.

Comment (MackinnonBuck) [inline test/Scaffolding/E2E_Test/Compiler/Resources/SharedViews/_ValidationScriptsPartial.cshtml:6]:
```suggestion
    <script src="https://ajax.aspnetcdn.com/ajax/jquery.validate/1.21.0/jquery.validate.min.js"
```

**Review by deepchoudhery — APPROVED** [review]
LGTM! will be creating dotnet-scaffold identity scaffolder soon as well! will add you to that PR for content check tyty.

**Pull request merged**